frontend/src/app/app.tsx

```diff
@@ -1,15 +1,36 @@
+import { actions as authActions } from '~/bundles/auth/store/auth.js';
 import {
+    Loader,
     Notifications,
-    PageLayout,
     RouterOutlet,
 } from '~/bundles/common/components/components.js';
+import { DataStatus } from '~/bundles/common/enums/enums.js';
+import {
+    useAppDispatch,
+    useAppSelector,
+    useEffect,
+} from '~/bundles/common/hooks/hooks.js';
 
 const App: React.FC = () => {
+    const dispatch = useAppDispatch();
+    const { dataStatus } = useAppSelector(({ auth }) => ({
+        dataStatus: auth.dataStatus,
+    }));
+
+    useEffect(() => {
+        void dispatch(authActions.loadUser());
+    }, [dispatch]);
+
+    if (
+        dataStatus !== DataStatus.FULFILLED &&
+        dataStatus !== DataStatus.REJECTED
+    ) {
+        return <Loader />;
```

Comment:
I am not sure if this should be wrapped in a modal. Perhaps in another ticket?

Reply:
No modal, just Loader